stl/inc/format

```diff
@@ -1352,6 +1391,15 @@ struct _Format_handler {
     }
 };
 
+// Generic formatter definition, the deleted default constructor
+// makes it "disabled" as per N4868 [formatter.requirements]/1.1.1

```

Comment:
I think a reference to [format.formatter.spec]/5 would be more appropriate, but feel free to ignore and/or change without rereview.